node/src/Pos/Generator/BlockEvent.hs

```diff
@@ -185,6 +185,7 @@ genBlocksInTree secrets bootStakeholders blockchainTree = do
             , _bgpBlockCount      = 1
             , _bgpTxGenParams     = txGenParams
             , _bgpInplaceDB       = True
+            , _bgpSkipNoKey       = False
             }
     -- Partial pattern-matching is safe because we specify
     -- blockCount = 1 in the generation parameters.
```

Comment:
I already wrote a rant that you've broken this logic (because I've spotted `catMaybes` in `genBlocks`), but now I see that `_bgpSkipNoKey = False` means that it's still valid. Please modify the comment to mention that now we rely on both `blockCount = 1` and `skipNoKey = False`.

Reply:
Where did you write it?

Reply:
I wrote it in this review but deleted when I realized that the code is correct.